macros/WebExtAPIOverview.ejs

```diff
@@ -2,22 +2,28 @@
 
 var output = "";
 var apiPages;
-var apiSummaryId = "api-summary";
 
-if ($0) {
-  apiPages = page.subpagesExpand($0);
-}
-else {
-  apiPages = page.subpagesExpand();
-}
+apiPages = page.subpagesExpand('/en-US/Add-ons/WebExtensions/API');
+
+var apiLinkLabel = mdn.localString({
+    "en-US": "API reference documentation",
+    "zh-CN": "API 参考资料"
+});
 
 for (var i=0; i < apiPages.length; i++) {
   var apiPage = apiPages[i];
   var apiPageUrlPieces = apiPage.url.split("/");
   var apiName = apiPageUrlPieces[apiPageUrlPieces.length-1];
+  var localepath = '/' + env.locale + '/docs/Mozilla/Add-ons/WebExtensions/API/' + apiName;
+  var apiPage_new = wiki.getPage(localepath);
+  if (apiPage_new.length != 0) {
```

Comment:
This doesn't work for testing for an empty object - ``apiPage_new.length`` returns ``undefined``, which is not equal to 0.  We have ECMA 5 available, so please use:

```
if (Object.keys(apiPage_new).length !== 0) {
```

https://stackoverflow.com/questions/679915/how-do-i-test-for-an-empty-javascript-object